packages/core/src/file/file.element.scss

```diff
@@ -7,3 +7,14 @@
 ::slotted(input) {
   display: none !important;
 }
+
+.input-container {
+  min-width: initial;
+}
+
+cds-button span {
```

Comment:
This may be okay for now. But the ask internally was for the name to be ellipsized in the middle such that `reallyLongFileName.png` would become `reallyLon...me.png` or something like that.